db/migrate/20220725205758_dissociate_withdrawn_mentors_from_induction_records.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class DissociateWithdrawnMentorsFromInductionRecords < ActiveRecord::Migration[6.1]
+  def change
+    InductionRecord.joins(:mentor_profile)
+                   .where(participant_profiles: { status: :withdrawn })
+                   .update_all(mentor_profile_id: nil)
+  end
+end
```

Comment:
I don't think we need this?

Reply:
Ok, this was when I thought we were going to nullify all the induction records mentored by the mentor deleted. 
I will remove it now

Reply:
Re-added the migration for current induction records still linked to withdrawn mentor profiles.